utils/src/api_schema/api_schema.json

```diff
@@ -452,8 +521,8 @@
       "properties": {
         "_ref":        { "$ref": "#/definitions/refInt" },
         "link":        { "$ref": "#/definitions/blockLink" },
-        "operations":  { "$ref": "#/definitions/blockOperations" },
-        "format":      { "$ref": "#/definitions/formatString" },
+        "operations":  { "$ref": "#/definitions/blockImageOperations" },
```

Comment:
Aren't these changes part of the previous commit? It probably breaks some test in the previous commit.

Reply:
nope, this is a result of adding video that we have to split operations block in two.